### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by lrose07**
add rubocop gem, fixed style infractions on whole repo

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -7,7 +9,7 @@
 Bundler.require(*Rails.groups)
 
 module Store
-  class Application < Rails::Application
+  class Application < Rails::Application # :nodoc:
```
Comment (kevellis124) [inline config/application.rb]: Is this where the comments should be? Not top of the file?

**Comment by lrose07** [comment]
The previous rubocop.yml wasn't fully relevant to Rails apps. I found the one that is recommended by the rails team and ran rubocop based on that. Full repo is 100% passing the rails style guide.

**Review by wkrause1 — APPROVED** [review]

File: Gemfile [hunk 2]
```diff
@@ -1,29 +1,31 @@
-source 'https://rubygems.org'
+# frozen_string_literal: true
+
+source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```
Comment (lrose07) [inline Gemfile]:
```suggestion
# test comment for GH suggestions
```

File: Gemfile [hunk 3]
```diff
@@ -1,29 +1,31 @@
-source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+# frozen_string_literal: true
 
-ruby '2.5.1'
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+# test comment for GH suggestions
```
Comment (lrose07) [inline Gemfile]:
```suggestion

```

**Pull request merged**